addons/website/views/snippets/s_timeline_images.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template name="Timeline Images" id="s_timeline_images">
+    <section class="s_timeline_images o_colored_level pt48 pb48">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 pb24 text-center" data-name="Heading">
+                    <h2 class="h3-fs">Evolving together</h2>
+                    <p class="lead">Highlight your history, showcase growth and key milestones.</p>
+                </div>
+            </div>
+            <div class="position-relative pt-3">
+                <div class="s_timeline_images_row position-relative d-flex flex-column" data-name="Milestone">
+                    <div class="o_dot_line position-absolute top-0 bottom-0 w-0 mb-1 border-start pe-none"/>
+                    <span class="o_dot o_not_editable position-absolute translate-middle-x rounded-circle pe-none text-o-color-1" contenteditable="false"/>
+                    <div class="s_timeline_images_content w-100 ps-4">
+                        <div class="o_text_columns">
+                            <div class="row">
+                                <div class="col-12 col-lg-4 pb16">
+                                    <small class="text-muted">13/06/2023</small>
+                                    <h3 class="h4-fs">First Milestone</h3>
+                                </div>
+                                <div class="col-12 col-lg-8 pb16">
+                                    <p>
+                                        <img src="/web/image/website.s_timeline_images_default_image_1" class="img img-fluid rounded" alt="" style="width: 100% !important;" />
```

Comment:
Extra space not needed 😉 Same for [L46](https://github.com/odoo/odoo/pull/184760/files#diff-012c786d5019caa8ef2218635c6cebe16a75fa5082a6b7e7eca639e4a054e557R46).
```suggestion
                                        <img src="/web/image/website.s_timeline_images_default_image_1" class="img img-fluid rounded" alt="" style="width: 100% !important;"/>
```